lib/axon_onnx/deserialize.ex

```diff
@@ -1383,6 +1383,12 @@ defmodule AxonOnnx.Deserialize do
           layer = Axon.layer([inp, min, max], fun, %{}, output_name, layer_op: :clip)
           updated_axon = Map.put(axon, output_name, layer)
           {updated_axon, used_params}
+
+        {%Axon{} = inp, %Nx.Tensor{} = min, %Nx.Tensor{} = max} ->
+          fun = fn x, _params -> Nx.clip(x, min, max) end
+          layer = Axon.layer(inp, fun, %{}, output_name, layer_op: :clip)
```

Comment:
`min` and `max` should be treated as parameters:

```elixir
fun = fn x, params -> Nx.clip(x, params[min_name], params[max_name]) end
```

And then ensure you add them to `used_params`!

Reply:
done -- i see that string literals are used as parameter names elsewhere... so i did that here too.